python/grass/docs/conf.py

```diff
@@ -12,15 +12,13 @@
 
 import sys
 import os
-from datetime import date
 import string
+from datetime import date
 from shutil import copy
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
-if not os.getenv("GISBASE"):
-    sys.exit("GISBASE not defined")
```

Comment:
This change of moving the check under has the sys.path.insert calls won't have the expected outcome. Starting at line 24, that environment variable is used. Having the check after having finished using it isn't equivalent.

Reply:
Thank you for pointing that out.  I've reconsidered the placement of the check and modified the code to ensure its effectiveness. I will push the changes to update the PR.